README.md

```diff
@@ -43,48 +49,66 @@ To programmatically check and authorize the self-hosted runner use the following
 
 Verify the signature and notarization of the binary
 
-`$ spctl -a -vvv -t install "$(brew --prefix)/bin/circleci-runner"`
+```shell
+$ spctl -a -vvv -t install "$(brew --prefix)/bin/circleci-runner"
+```
 
 Accept the Apple notarization ticket
 
-`$ sudo xattr -r -d com.apple.quarantine "$(brew --prefix)/bin/circleci-runner"`
+```shell
+$ sudo xattr -r -d com.apple.quarantine "$(brew --prefix)/bin/circleci-runner"
+```
 
 #### Stopping and Restarting the Self-Hosted Runner
 
-To start the self-hosted runner the first time after it has been installed and configured run to enable the service:
+To start the self-hosted runner the first time after it has been installed and configured run to bootstrap the service as follows:
 
-```$ launchctl bootstrap gui/`stat -f %u` $HOME/Library/LaunchAgents/com.circleci.runner.plist```
+```shell
+$ launchctl bootstrap gui/$(id -u) $HOME/Library/LaunchAgents/com.circleci.runner.plist
+$ launchctl enable  gui/$(id -u)/com.circleci.runner
+$ launchctl kickstart -k gui/$(id -u) /com.circleci.runner
+```
 
-##### Running in Headless Mode
-For running the self-hosted runner in a headless or non-GUI session, use the user domain in the launchctl bootstrap command as follows:
+##### Running in a non-GUI session
+For running the self-hosted runner in a headless or non-GUI session, the `.plist` file should be moved to `/Library/LaunchAgents`, which is for per-user agents provided by the administrator, and ensures that the service will run on reboot:
 
-```$ launchctl bootstrap user/`stat -f %u` $HOME/Library/LaunchAgents/com.circleci.runner.plist```
+```shell
+$ sudo mv $HOME/Library/LaunchAgents/com.circleci.runner.plist /Library/LaunchAgents/
+```
 
-Then start the service with: 
+Finally, use the user domain in the `launchctl` bootstrap command as follows:
 
-`$ launchctl load $HOME/Library/LaunchAgents/com.circleci.runner.plist`
```

Comment:
The `launchctl <load|unload>` commands are [considered legacy](https://ss64.com/mac/launchctl.html), and we saw some arcane errors related to them (such as `Load failed: 5: Input/output error`), particularly when trying to install runner during non-GUI sessions on `mac2.metal` EC2 instances. This PR moves us off these commands.